ci-operator/step-registry/osd-ccs/fleet-manager/osd-ccs-fleet-manager-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: osd-ccs-fleet-manager
```

Comment:
It should be put under the sc folder, and the workflow is `osd-ccs-fleet-manager-sc`